include/generic_sort.h

```diff
@@ -41,7 +45,12 @@ std::string to_smtlib(SortKind);
 class GenericSort : public AbsSort
 {
  public:
-  GenericSort(SortKind sk); 
+  GenericSort(SortKind sk);
+  // This constructor is only for datatype sorts.
+  GenericSort(SortKind sk, std::string sort_name);
+  // Only for placeholder sorts used for inductive datatypes. name
```

Comment:
```suggestion
  // Only for placeholder sorts used for datatypes. name
```
`Inductive`is sometimes used to refer to a subset of datatypes, but here we cover also non-inductive datatypes (like records).